### ISSUE TITLE
Question about the Sign of SIMSE Loss

### ISSUE THREAD

**Issue by MaxLiu245** [fungtion/DSN#9]
Hi, I was reading the DSN paper and saw you codes here. Nice codes!

But here I was wondering whether the loss codes in `train.py`'s line 189 & 245 should be `loss -= source_simse` and `loss -= target_simse`. To confirm this, I read the original `scale-invariant mean squared error` paper, especially the eqn. 4 and made this conclusion.

I thought if so, the acc can be higher, and get close to DSN's result~

**Comment by BX-xb** [comment]
Hi, do you have the code of the original paper? It seems that the github link in the paper cannot be entered

**Comment by codenie** [comment]
The official implementation of the DSN model is in the archive brach, the link is as follows:
https://github.com/tensorflow/models/tree/archive/research/domain_adaptation


> Hi, do you have the code of the original paper? It seems that the github link in the paper cannot be entered

**Comment by codenie** [comment]
In addition to the sign, the `log` should also be used, according to the equation of SIMSE:
![image](https://user-images.githubusercontent.com/31645754/110463244-9edff480-810c-11eb-80fe-b5ca22b907fb.png)
where ![image](https://user-images.githubusercontent.com/31645754/110463264-aacbb680-810c-11eb-90d6-4d1f25d7ba3a.png)
Is this right?
